src/subcommand/wallet/create.rs

```diff
@@ -17,10 +17,6 @@ pub(crate) struct Create {
 
 impl Create {
   pub(crate) fn run(&self, options: Options) -> Result {
-    if !(self.name == "ord" || self.name.starts_with("ord-")) {
```

Comment:
Commenting here: we should remove struct Create and take the wallet name from the options.